src/app/client/src/app/modules/groups/components/group-header/group-header.component.ts

```diff
@@ -223,6 +223,7 @@ export class GroupHeaderComponent implements OnInit, OnDestroy {
         type: 'Group'
       }
     ];
+    this.discussionTelemetryService.parentRouteUrl = window.location.href;
```

Comment:
Why don't we use NavigationHelperService?

discussionTelemetryService should have logic specific to only telemetry

Reply:
Done